scripts/install

```diff
@@ -60,8 +60,10 @@ pushd "$install_dir"
 	curl https://sh.rustup.rs -sSf | ynh_exec_warn_less ynh_exec_as $app sh -s -- -y
 	export PATH="$install_dir/.cargo/bin:$PATH"	
 	ynh_use_nodejs
-	corepack enable
-	corepack prepare pnpm@latest --activate
+	env $ynh_node_load_PATH corepack enable
+	env $ynh_node_load_PATH corepack prepare pnpm@latest --activate
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack enable
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack prepare pnpm@latest --activate
 	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH pnpm install
```

Comment:
```suggestion
	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH corepack pnpm install
```

I was reading from here that `corepack` must be used along with `pnpm install`

cf -> https://www.raulmelo.me/en/blog/making-the-switch-to-pnpm